src/layouts/Layout.astro

```diff
@@ -1,22 +1,15 @@
 ---
-interface Props {
-	title: string;
-}
-
-const { title } = Astro.props;
+import NavBar from '../components/NavBar.astro';
 ---
 
-<!doctype html>
-<html lang="en">
-	<head>
-		<meta charset="UTF-8" />
-		<meta name="description" content="Astro description" />
-		<meta name="viewport" content="width=device-width" />
-		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
-		<meta name="generator" content={Astro.generator} />
-		<title>{title}</title>
-	</head>
-	<body>
-		<slot />
-	</body>
+<html>
+  <head>
+    <title>My Astro Site</title>
```

Comment:
Look at the original layout code: get the title from Astro.props and use that, not My Astro Site.